perllib/FixMyStreet/Roles/CobrandPay360.pm

```diff
@@ -296,7 +296,7 @@ sub waste_get_current_garden_sub {
 
     my $garden;
     for my $service ( @$services ) {
-        if ( $service->{ServiceId} == $self->garden_waste_service_id ) {
+        if ( $service->{ServiceId} == $self->garden_bin_service_id ) {
```

Comment:
Not using Pay360 for Kingston, so seemed not as important - it also won't work because it's assuming service ID rather than task ID, for example, wasn't sure how to deal. Should note it though, unless being done differently in your work anyway (which I guess will have to account for it somehow).